domain/resource/service/resource.go

```diff
@@ -305,6 +316,24 @@ func (s *Service) GetResourcesByApplicationID(ctx context.Context, applicationID
 	return s.st.GetResourcesByApplicationID(ctx, applicationID)
 }
 
+// ExportResources retrieves resources associated with a specific
+// application name. Returns a slice of resources or an error if the operation
+// fails.
+//
+// The following error types can be expected to be returned:
+//   - [resourceerrors.ApplicationNotFound] is returned if the application name
```

Comment:
question: what is the purpose of returning this error? Can the caller recover from it and try again?

Reply:
I just put the comment there out of habit, have removed it.